SPECS/icu/icu.spec

```
@@ -60,6 +64,10 @@ make -C icu/icu4c/source DESTDIR=%{buildroot} install
 %{_libdir}/pkgconfig/*.pc
 
 %changelog
+* Thu Feb 1 2024 corvus-callidus <108946721+corvus-callidus@users.noreply.github.com> - 72.1.0.3-1
```

nit: We often use 0-padded two-digit numbers for the day of the month (also might be worth updating the date).
```suggestion
* Thu Feb 05 2024 corvus-callidus <108946721+corvus-callidus@users.noreply.github.com> - 72.1.0.3-1
```